hawc/apps/assessment/forms.py

```diff
@@ -40,7 +41,11 @@ class Meta:
         model = models.Assessment
 
     def __init__(self, *args, **kwargs):
+        self.user = kwargs.pop("user", None)
```

Comment:
is it possible to remove the default None in the kwargs pop? If so, then it makes it easier to reason about since you know self.user could never be None (ie, only one code path)

Reply:
Ha I'm not sure... I think you wrote this code!

Reply:
I'll take a look at it

Reply:
It looks like it breaks the test_edit_permissions if I remove it

Reply:
ok, silly me